docs/reference/cli.md

```diff
@@ -1790,6 +1790,18 @@ uv lock [OPTIONS]
 <p>To view the location of the cache directory, run <code>uv cache dir</code>.</p>
 
 <p>May also be set with the <code>UV_CACHE_DIR</code> environment variable.</p>
+</dd><dt><code>--check</code></dt><dd><p>Check if the lockfile is up-to-date.</p>
+
+<p>Asserts that the <code>uv.lock</code> would remain unchanged after a resolution. If the lockfile is missing or needs to be updated, uv will exit with an error.</p>
+
+<p>Equivalent to <code>--locked</code>.</p>
+
+<p>May also be set with the <code>UV_LOCKED</code> environment variable.</p>
```

Comment:
The env var / CLI name mismatch here is a bit strange, but oh well.

Reply:
Yeah, I noticed that too but hopefully the "Equivalent to" call-out above is helpful. `UV_LOCK_CHECK` doesn't really make sense to set via the environment anyway.